checkov/terraform/runner.py

```diff
@@ -147,6 +152,18 @@ def run(
 
         report = remove_duplicate_results(report)
 
+        if bc_integration.bc_api_key and any(framework in runner_filter.framework for framework in ("all", CheckType.SCA_IMAGE)):
```

Comment:
why does the handling of image-referencer happen in terraform-runner? isn't it more related to sca-image scan?

Reply:
ok now I saw that the resources are located in the scanned terraform files. so it make sense =]